M2/Macaulay2/m2/shared.m2

```diff
@@ -24,6 +24,10 @@ isEmpty = method(TypicalValue => Boolean)
 isSmooth = method(TypicalValue => Boolean, Options => true)
 isVeryAmple = method(TypicalValue => Boolean, Options => true)
 
+isNormal = method()
+
+normalCone = method(Options => true)
```

Comment:
These two should be added in `packages/Macaulay2Doc/shared.m2`, ideally with references to `IntegralClosure::isNormal` and other packages using these symbols.

Reply:
doc added